common-rendering/src/components/keyEvents.tsx

```diff
@@ -201,10 +203,18 @@ const timeStyles = (supportsDarkMode: boolean): SerializedStyles => css`
 `;
 
 const ListItem = ({ keyEvent, theme, supportsDarkMode }: ListItemProps) => {
+	const time = timeAgo(new Date(keyEvent.time).getTime());
+
 	return (
 		<li css={listItemStyles(supportsDarkMode)}>
 			<div css={timeTextWrapperStyles}>
-				<time css={timeStyles(supportsDarkMode)}>{keyEvent.time}</time>
+				<time
+					dateTime={keyEvent.time}
+					data-relativeformat="short"
```

Comment:
Is this part of the `<time>` interface? I've not seen it before.

Reply:
It's just what the `updateTimeElements` function uses to determine the verbosity of the `timeAgo` string, I don't think it's specific to `time` though.

Reply:
`data-relativeformat` is a convention first establish at the Guardian at least **_9_** years ago in [PR #38](https://github.com/guardian/frontend/pull/38), today we're on number 24179 - so as Guardian code conventions go, it's probably the oldest! (But I could be wrong - who knows what's out there...) It's also still in use today [on Frontend](https://github.com/guardian/frontend/search?q=data-relativeformat). 

See: https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/ui/relativedates.js

The `timeAgo` lib was built to carry this convention on into DCR and to other repos in the department that need relative time. We actually decided to modify the api slightly, ending up on the `verbose` boolean you see. This was carefully worked out in [this discussion](https://github.com/guardian/libs/pull/134) leading the transformation (you see in this PR)

```ts
				switch (relativeFormat) {
					case "short":
					case "med":
						newTime = timeAgo(new Date(absoluteTime).getTime(), {
							verbose: false,
						});
						break;
					case "long":
						newTime = timeAgo(new Date(absoluteTime).getTime(), {
							verbose: true,
						});
						break;
				}
```